src/inc/litectx.h

```diff
@@ -14,7 +14,10 @@
 #include "hclib_common.h"
 #include "fcontext.h"
 #include <string.h>
+
+#ifdef HAVE_SYS_MMAN_H
```

Comment:
I'm assuming we have the same situation here as with aio.h? HAVE_SYS_MMAN_H will be automatically set if sys/mman.h is findable on a given system, by autoconf?

Reply:
Yes. it is set inside  hclib-install/include/hclib_config.h when HClib is build.